### ISSUE TITLE
Open Seattle in 2016

### ISSUE THREAD

**Issue by sethvincent** [openseattle/discuss#25]
What would you like Open Seattle to accomplish in 2016?

How might you contribute to Open Seattle next year?

What are the most important issues in Seattle you think civic tech projects should focus on next year?

**Comment by chrismetcalf** [comment]
A few suggestions:

- More collaboration with other community groups. This can be as simple as having them come talk about an issue that has been or they think may be addressed with tech, or as deep as building a team to help them solve some of those issues
- More frequent project status updates and presentations. I know people are working on cool stuff, we just need to be more vocal about it!
- More education. I think the Node School stuff is awesome, and we should do more of it. We also should involve bootcamps like Ada and Galvanize more
- Host a "Ship Camp". Everybody brings their half-finished projects and we work together to get them out the door.

**Comment by sethvincent** [comment]
@chrismetcalf Ship Camp yeeessss. :ship: :tent:

**Comment by sethvincent** [comment]
Some things I'd like to work on:

- "More collaboration with other community groups." <- this
- Make the openseattle.org blog super active
- A regular cycle for creating & publishing small visualization/explainer projects
- Create and improve API services for building Seattle-focused apps
- Better documentation of:
  - Open Seattle processes & policies
  - tasks & projects newcomers can take on
  - existing & completed projects
  - resources & tools for civic tech
  - what even is civic engagement, local government, & community organizing (and where does technology fit in to this mess because sometimes it doesn't)

**Comment by loganfranken** [comment]
+1 for "A regular cycle for creating & publishing small visualization/explainer projects" and "More frequent project status updates and presentations"

For me, personally, the more ideas and examples I can see of what's possible and what can be done with open data in Seattle, the better.

**Comment by JohnTigue** [comment]
Elections political contributions data viz projects. The stuff on the City's site is raw and way too hard to use for quick digging on simple questions.

**Comment by ptressel** [comment]
@chrismetcalf -- Double-plus extra special LGTM for Ship Camp!  Got any particular community organizations in mind?  (I have contacts with assorted emergency response and mapping volunteer groups, but not so much the responder agencies that could use help.)

@JohnTigue -- Does it look like some preliminary data cleaning, reformatting, and restructuring might make it easier for folks to do queries?

I suggested to Seattle GiveCamp folks that Open Seattle could be a nice way to continue their work.  Haven't yet gotten my peeps (working on a project for Facing Homelessness) to show up...

**Comment by chrismetcalf** [comment]
@ptressel - Perhaps @candacefaber would have some ideas for good community organizations we could partner with.

**Comment by candacefaber** [comment]
How did you know? I have some ideas, but I recommend starting with issues
rather than orgs and convening around areas of interest. Do we know what
specific vectors people are excited about? (Arts & culture, housing, etc?)

@ptressel <https://github.com/ptressel> - Perhaps @candacefaber
<https://github.com/candacefaber> would have some ideas for good community
organizations we could partner with.

—
Reply to this email directly or view it on GitHub
<https://github.com/openseattle/discuss/issues/25#issuecomment-162412910>.

**Comment by chrismetcalf** [comment]
@candacefaber I hear you might be dropping by @openseattle soon, so maybe we can have a group discussion about what people would be interested in.

**Comment by candacefaber** [comment]
That we will! I'll be there for the first meetup in January. Keep yours
eyes out for a new thread on the civic tech "best of 2015" too - doing a
writeup and want to make sure I didn't miss anybody!

**Comment by JohnTigue** [comment]
Continue working on the ISP tracking project.

Deploy the same codebase that the City of Seattle is deploying for ISP performance tracking but do it outside the City, ourselves. The City's deploy is configured (as is appropriate for a city) with a level of data anonymization which protects citizens privacy. For example, a particular census block group will only light up in reports once there are at least 5 recorded tests.

The goal of OpenSeattle's deploy (of the same codebase) would be to have a version that has no data anonymization. Users would be able to registers and track their speed tests over time to their account (the City's deploy intentionally does not have accounts). The personal data would be sharable, associated with the user. (Clearly, we would need to make sure that users understand that their data is not anonymous.)

Then once the stock deploy is up and running, we could extent the codebase. We could come up with our own analytics for reports on the data. We could add the ability to track interactions with the ISPs, perhaps even using the corporations' own incident numbers in our databaase. It would be valuable to collectively record our interactions with the ISPs.

More immediately there is an need to generate awareness. Perhaps an email campaign for folks to invite their friends to run a speed test (and perhaps add a widget-ized tester&map to their start page, etc.). Or monthly emails to request that folks run another test. This could be on [AWS's SES](https://aws.amazon.com/ses/) (this would be written in JavaScript), or MailChimp.

Federation of data across multiple deploys would be a valuable enhancement. That is, how can a deploy by Portland and Deploy by Seattle and others be merged together?

Basically, keep it going on this front.

**Comment by candacefaber** [comment]
:) awesome stuff. Thanks again for helping with user testing. No rush on the notes - if you can get them to me by Monday am I can work them through the process. Given the level of publicity the City can get from this effort, I want to make sure that what we deploy is as good as we can make it, so I'm incredibly thankful for all your hard work to make it so!

Sent from my iPhone

On Jan 7, 2016, at 9:42 PM, John Tigue <notifications@github.com<mailto:notifications@github.com>> wrote:


Continue working on the ISP tracking project.

Deploy the same codebase that the City of Seattle is deploying for ISP performance tracking but do it outside the City, ourselves. The City's deploy is configured (as is appropriate for a city) with a level of data anonymization which protects citizens privacy. For example, a particular census block group will only light up in reports once there are at least 5 recorded tests.

The goal of OpenSeattle's deploy (of the same codebase) would be to have a version that has no data anonymization. Users would be able to registers and track their speed tests over time to their account (the City's deploy intentionally does not have accounts). The personal data would be sharable, associated with the user. (Clearly, we would need to make sure that users understand that their data is not anonymous.)

Then once the stock deploy is up and running, we could extent the codebase. We could come up with our own analytics for reports on the data. We could add the ability to track interactions with the ISPs, perhaps even using the corporations' own incident numbers in our databaase. It would be valuable to collectively record our interactions with the ISPs.

More immediately there is an need to generate awareness. Perhaps an email campaign for folks to invite their friends to run a speed test (and perhaps add a widget-ized tester&map to their start page, etc.). Or monthly emails to request that folks run another test. This could be on AWS's SES<https://aws.amazon.com/ses/> (this would be written in JavaScript), or MailChimp.

Federation of data across multiple deploys would be a valuable enhancement. That is, how can a deploy by Portland and Deploy by Seattle and others be merged together?

Basically, keep it going on this front.

**Comment by JohnTigue** [comment]
@ptressel Not sure what the solution is but for a certain topic I wanted to figure out which corporations and interest groups were donating to which candidates and is was way too hard.

**Comment by candacefaber** [comment]
Oooh, you should check in with Sunlight Foundation. Their Influence Explorer tool uses FEC data. It would be cool to work with them on a local version. 

The Seattle Ethics and Elections Commission currently publishes a ton of information, but it's not in machine-readable format. Let me know if you are interested. It would be so cool if we could have a local version of Influence Explorer, but first we will have to do a lot of work to get the SEEC data into shape.

If anyone decides to pursue that in earnest, let me know and I'll talk to that office - or you can reach out directly, if you'd rather!
